### ISSUE TITLE
Put documentation on website

### ISSUE THREAD

**Issue by batterseapower** [batterseapower/pinyin-toolkit#74]
It's all very well having some files in the docs/ directory, but it's kind of inaccessible for users. Wouldn't it be better to format that stuff as HTML and put it on the web, with a link in the "Help" menu (which we already have)?

**Comment by Nick3C** [comment]
yes it would. Let's transition it now in a raw form and tidy it up later shall we?

Still need a license file (long) but readme can be merged with Pinyin Toolkit.py and rest put on the web.

**Comment by Nick3C** [comment]
I've moved them to the wiki, but need to reformat them so they look nice.

**Comment by batterseapower** [comment]
I massively cleaned up the docs on the wiki. I still need to make a small change to the site to point at the Wiki rather than the HTML docs - I think I'll just keep the HTML around as a 1-page high-level description of what the TK does.
